docs/core/tools/cli-msbuild-architecture.md

```diff
@@ -13,7 +13,7 @@ ms.assetid: 7fff0f61-ac23-42f0-9661-72a7240a4456
 
 # High-level overview of changes in the .NET Core tools
 
-This document will describe in high-level the changes that moving from *project.json* to MSBuild and *.csproj* project system bring. It will outline the new way the tooling is layered all-up and which new pieces are available and what is their place in the overall picture. After reading this article, you should have a better understanding of all of the pieces that make up .NET Core tooling after moving to MSBuild and *.csproj*. 
```

Comment:
I don't think versions of .Net Core are relevant here, only .Net Core SDK (I think that's the right name to use here, not ".Net Core Tools") and VS are affected by the change of the project format.